ci-operator/config/jianzhangbjz/learn-operator/jianzhangbjz-learn-operator-master.yaml

```diff
@@ -0,0 +1,76 @@
+base_images:
+  base:
+    name: "4.7"
+    namespace: ocp
+    tag: base
+binary_build_commands: make build
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.14
+images:
+- dockerfile_path: build/Dockerfile.ci
+  from: base
+  inputs:
+    bin:
+      paths:
+      - destination_dir: .
+        source_path: /tmp/learn-operator
+      - destination_dir: .
+        source_path: /tmp/bin
+  to: learn-operator
+operator:
+  bundles:
+  - context_dir: manifests
+    dockerfile_path: bundle.Dockerfile
+  substitutions:
+  - pullspec: quay.io/olmqe/learn-operator:v1
+    with: learn-operator
+promotion:
+  name: learn
+  namespace: ocp
+releases:
+  latest:
+    release:
+      channel: stable
+      version: "4.6"
```

Comment:
This makes zero sense. You base your operator on 4.7 base, but want to test it on stable 4.6. Why?

Reply:
I read this example: https://docs.ci.openshift.org/docs/architecture/ci-operator/#testing-with-an-existing-openshift-release, it says `latest.release` is `# references a version released to customers`, the 4.7 hasn' been released to customers, and that example uses 4.4 as latest, 4,5 as previous. So, I thought here should be `4.6`.
```console
latest:
    release:          # references a version released to customers
      channel: stable # configures the release channel to search
      version: "4.4"
  previous:
    candidate:
      product: ocp
      architecture: amd64
      stream: nightly     # specifies a candidate release stream
      version: "4.5"
      relative: 1         # resolves to the Nth latest payload in this stream
```

Reply:
It's an *example* that shows different possibilities how to specify a release, not a snippet to be copy-pasted.